flocker/node/agents/cinder.py

```diff
@@ -530,7 +530,7 @@ def list_volumes(self):
         http://docs.rackspace.com/cbs/api/v1.0/cbs-devguide/content/GET_getVolumesDetail_v1__tenant_id__volumes_detail_volumes.html
         """
         flocker_volumes = []
-        for cinder_volume in self.cinder_volume_manager.list(detailed=False):
+        for cinder_volume in self.cinder_volume_manager.list(detailed=True):
```

Comment:
Is there a case where we do not always need `detailed=True`? I seem to remember the discusssion that Cinder v1 would pass with `detailed=False`, thus the detailed information is not needed? maybe im making this up, but making sure we're not requested uneeded information that may slow down the process being that every volume may have more information.